modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala

```diff
@@ -107,9 +107,7 @@ class SeqexecEngine(settings: SeqexecEngine.Settings) {
         obsId <- safeGetObsId(id)
         _ <- systems.odb.obsAbort(obsId, e.msg)
       } yield ()
-      case (EventSystem(Executed(id)), st) if st.sequences.get(id).map(
-        _.status === SequenceState.Idle
-      ).getOrElse(false) => for {
+      case (EventSystem(Executed(id)), st) if st.sequences.get(id).exists(_.status === SequenceState.Idle) => for {
```

Comment:
I can't do `Event.executed(id)` here as we are inside a pattern match here, right?